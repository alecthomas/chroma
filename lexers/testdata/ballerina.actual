import ballerina/http;

endpoint http:Listener listener {
    port: 9090
};
