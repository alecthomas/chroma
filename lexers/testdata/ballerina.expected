[
  {"type":"KeywordNamespace","value":" "},
  {"type":"NameNamespace","value":"ballerina"},
  {"type":"Operator","value":"/"},
  {"type":"Name","value":"http"},
  {"type":"Operator","value":";"},
  {"type":"Text","value":"\n\n"},
  {"type":"KeywordDeclaration","value":"endpoint"},
  {"type":"Text","value":" "},
  {"type":"NameLabel","value":"http:"},
  {"type":"Name","value":"Listener"},
  {"type":"Text","value":" "},
  {"type":"Name","value":"listener"},
  {"type":"Text","value":" "},
  {"type":"Operator","value":"{"},
  {"type":"NameLabel","value":"\n    port:"},
  {"type":"Text","value":" "},
  {"type":"Name","value":"9090"},
  {"type":"Text","value":"\n"},
  {"type":"Operator","value":"};"},
  {"type":"Text","value":"\n"}
]
